src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntityLivingBase.java

```diff
@@ -933,4 +933,8 @@ public void supplyVanillaManipulators(List<DataManipulator<?, ?>> manipulators)
         return ProjectileLauncher.launch(checkNotNull(projectileClass, "projectile class"), this, checkNotNull(velocity, "velocity"));
     }
 
+    public void setElytraFlying(boolean value) {
+        setFlag(7, value);
```

Comment:
:-1: for magic number

Reply:
Well that's how they set it, not sure of a better way since they don't have an explicit method besides that.

Reply:
You could add a constant to replace it. e.g. 

`public static final int FLAG_ELYTRA = 7;

Reply:
DataConstants!